src/test/java/teammates/test/pageobjects/InstructorCommentsPage.java

```diff
@@ -264,6 +264,15 @@ private boolean isAllCommentPanelBodyVisibilityEquals(boolean isVisible) {
         return true;
     }
     
+    public boolean isStudentCommentsPanelBodyVisible() {
+        return getStudentCommentsPanelBody().isDisplayed();
+    }
+    
+    private WebElement getStudentCommentsPanelBody() {
+        WebElement studentCommentsPanel = browser.driver.findElement(By.className("student-comments-panel"));
+        return studentCommentsPanel.findElement(By.className("panel-body"));
```

Comment:
Would it be ok to simply return `browser.driver.findElement(By.cssSelector(".student-comments-panel .panel-body"));`